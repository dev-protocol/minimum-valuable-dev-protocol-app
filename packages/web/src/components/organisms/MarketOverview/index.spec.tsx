import React from 'react'
import { render } from '@testing-library/react'
import { MarketOverview } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${MarketOverview.name}`, () => {
  test('Snapshot', () => {
    const component = render(<MarketOverview />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
