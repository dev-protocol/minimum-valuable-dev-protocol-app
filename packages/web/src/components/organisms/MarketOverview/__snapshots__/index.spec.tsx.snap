// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarketOverview Snapshot 1`] = `
<body>
  <div>
    <div
      style="max-width: 680px; margin-right: auto; margin-left: auto;"
    >
      <div
        class="ant-row"
        style="margin: 82px 0px;"
      >
        <div
          class="ant-col ant-col-24"
        >
          <div>
            <h2
              style="font-size: 36px; line-height: 48px; color: rgb(47, 128, 237);"
            >
              npm
            </h2>
            <div
              style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
            >
              Marketize the number of downloads of public npm packages.
            </div>
            <div
              style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
            >
              Authentication: 
              Your npm package
            </div>
            <div
              style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
            >
              Calculation: 
              Number of downloads
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
